migrations/2025-07-15-081447_group-follow/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE community_community_follow (
+    community_id int REFERENCES community (id) ON UPDATE CASCADE ON DELETE CASCADE NOT NULL,
+    follower_id int REFERENCES community (id) ON UPDATE CASCADE ON DELETE CASCADE NOT NULL,
+    PRIMARY KEY (community_id, follower_id)
```

Comment:
Add a `published_at timestamptz not null default now` column.

Reply:
For what? This data isnt read nor displayed anywhere.

Reply:
For accounting purposes, or just in case we need to migrate and it needs some orderable key to work from, and a lot of other reasons I can't think of rn. We should always include a published column for every source data table.